exchange/exchange_test.go

```diff
@@ -2288,6 +2298,21 @@ func runSpec(t *testing.T, filename string, spec *exchangeSpec) {
 		}
 
 	}
+
+	if spec.BidValidationEnforcement == config.ValidationEnforce {
```

Comment:
`spec.BidValidationEnforcement` does not appear to be set anywhere so this section does not run. I would expect it to be set either in the json file or maybe in line 2201.

There are actually some differences between the ext in json and the one output, that were uncovered when comparing `spec.Response.Ext` when present in a PR I am working on.

Reply:
Thank you for catching this! I used to have an old way of setting the validation enforcement, and then I updated how I set the object but forgot to update this line. I am going to push a change to address this!